test/test-cases/regression/rule-920450.json

```diff
@@ -0,0 +1,87 @@
+[
+  {
+    "enabled":1,
```

Comment:
Since this is a pull request that was well described in a issue i miss the **github_issue** tag.

As listed -
https://github.com/SpiderLabs/ModSecurity/blob/785958f9b5089b918c7d054cbcc2fe4a3c7b3788/test/test-cases/regression/issue-960.json#L7-L9

Reply:
Thanks for info. The last commit contains both 'url' and 'github_issue' keys, in both test cases.

Note: it would be good to make some wiki page about your expectations for contributors. I'm sure these keywords are part of the regression test structure, but after 17 commits I've met with them first.